samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -758,7 +758,7 @@ class SamzaContainer(
       if (containerListener != null) {
         containerListener.afterStart()
       }
-      metrics.containerStartupTime.update(System.nanoTime() - startTime)
+      metrics.containerStartupTime.set(System.nanoTime() - startTime)
```

Comment:
Let's change this to use currentTimeMillis as well. I think we're pretty far from sub-ms startup times :)

Reply:
done